sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuFileFormatDataWriter.scala

```diff
@@ -400,8 +448,6 @@ class GpuDynamicPartitionDataWriter(
         outputColumns.close()
       }
 
-      splits.safeClose()
-
       if (cbKeys != null) {
```

Comment:
We could file a follow-up ticket to clean the null pointers (using `withResource` instead ).